src/RequestSpec.ts

```diff
@@ -36,9 +36,6 @@ export interface TagValue {
   operator?: string;
 }
 
-// TODO: is unused.
```

Comment:
could/should we try to use this?
same thing for graph_type: we know this can only be `graph` or `metric`?
perhaps you can give this a try. if too many errors pop up, we can remove it.

Reply:
btw: you added this type as "Presentation" in the commit before this commit. perhaps we can rename it here? should we rename "presentation" to aggregation, and use this type instead?

Reply:
The name seems to have been carried over from the WebApi, but Aggregation make a lot more sense to me.